dotnet/src/SemanticKernel.Core/Functions/KernelFunctionFromPrompt.cs

```diff
@@ -422,10 +422,6 @@ private void CaptureUsageDetails(string? modelId, IReadOnlyDictionary<string, ob
             jsonObject.TryGetProperty("CompletionTokens", out var completionTokensJson) &&
             completionTokensJson.TryGetInt32(out int completionTokens))
         {
-            logger.LogInformation(
-                "Prompt tokens: {PromptTokens}. Completion tokens: {CompletionTokens}.",
-                promptTokens, completionTokens);
```

Comment:
Is this safe to remove? Does it mean that this information won't be logged anymore?

Reply:
This information is is already logged at the connector level as I answered above.

https://github.com/microsoft/semantic-kernel/blob/4cafb9519879e28d13e18673206001cd06dcba51/dotnet/src/Connectors/Connectors.OpenAI/Core/ClientCore.ChatCompletion.cs#L887